walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -99,9 +104,11 @@ public void everyAttendeeIsConsidered() {
     // Have each person have different events. We should see two options because each person has
```

Comment:
It says two options here, but the diagram shows three options. Is this "two" a typo?